test_plugins/validate.py

```diff
@@ -9,7 +11,7 @@ def valid_interface(configuration):
     * interface name is 15 characters or less
     * interface name does not start with a number
     """
-    for config_section in [s for s in ("bonds", "interfaces") if s in configuration.keys()]:
+    for config_section in [s for s in ("bonds", "interfaces") if s in viewkeys(configuration)]:
```

Comment:
I thought the view returned by `dict.keys()` in python3 supports the `in` membership test, so why is this compatibility function needed?